hal/halx86/smp/i386/spinup.c

```diff
@@ -26,8 +25,9 @@ extern PVOID HalpAPEntryData;
 extern PVOID HalpAPEntry32;
 extern PVOID HalpAPEntry16End;
 extern HALP_APIC_INFO_TABLE HalpApicInfoTable;
+extern PROCESSOR_IDENTITY HalpProcessorIdentity[MAXIMUM_PROCESSORS];
 
-ULONG HalpStartedProcessorCount = 1;
+ULONG HalpStartedProcessorCount = 0;
```

Comment:
```suggestion
ULONG HalpStartedProcessorCount;
```

Reply:
we have to pray that the BSS gets correctly zeroed out :joy:

Reply:
There are zero counted processors 
not NULL counted processors

Reply:
I don't understand: aren't globals automatically zeroed?